packages/react-magma-dom/src/components/Datagrid/Datagrid.stories.tsx

```diff
@@ -337,18 +343,23 @@ const defaultArgs = {
 };
 
 export const Default = Template.bind({});
-Default.args = defaultArgs;
+Default.args = { ...defaultArgs, tableTitle: 'Default' };
+
+export const TitleWithNode = Template.bind({});
+TitleWithNode.args = { ...defaultArgs, tableTitle: <Heading level={1}>Title with node</Heading> };
```

Comment:
We could do something like this to exclude tableTitle from this story in order for it to not get replaced
`TitleWithNode.parameters = { controls: { exclude: ['tableTitle'] } };`